index.Rmd

```diff
@@ -429,6 +435,144 @@ You can also browse the history of a repository from your Git hosting service, a
+-->
+
+# *RStudio* setup {#gettingstarted}
+For this course you can use RStudio to write and run scripts. You can access it from e.g. *Applications* → *Programming* → *RStudio* menu.
+In case you are working on your own computer and would like to know how to install *R* and *RStudio*, see the [RStudio website](https://www.rstudio.com/products/rstudio/download/).
+
+## Getting started with *RStudio* and *R* on your own computer.
+
+If you are new to RStudio take a look at the following summary YouTube video about how to use RStudio: [Intro to RStudio (6 min)](https://www.youtube.com/watch?v=FIrsOBy5k58). In it you will learn how to navigate the RStudio environment and even run some code. This will be helpful for later in the course.
+
+Additionally, the first time you open RStudio it's helpful to change some global settings. To do this go to *Tools* → *Global Options...* in the top menu bar of RStudio. The following box should appear. Set the settings to match that of the screenshot, namely, *Never save the workspace to .RData file on exit* and uncheck the boxes under *History*.
+<center> 
+![Global Options](figs/global_options.png)
+<center>
+
+<!-- ### Set Your Working Directory
+
+Let's do some basic set up first.
+
+- Create a git repository which will be your working directory e.g. `Lesson1`
+- Create an R script within that directory
+- Set your working directory to `Lesson1`
+- Create a `data` directory within your working directory
+
+In RStudio you can set your working directory this way, if you have saved your R script to your working directory: 
+- In the RStudio menu bar, go to *Session* → *Set Working Directory* → *To Source File Location*
+
+<center>
+![Set working directory → to source file location](figs/setwd.png)
+<center>
+
+
+```{block, type="alert alert-danger"}
+**Warning**: The use of `setwd()` is not a good practice in scripts without a very, very good reason. The users are aware of the environment and working directories and scripts should not try to outsmart the users.
+*Never set either of the following in your script! This would make the script not reproducible by others!*
+    setwd("/some/path/on/your/computer")
+    setwd("../some/path/outside/your/repository")
+```
+
+
+```{block type="alert alert-info"}
+**Protip**: You can use relative paths to load your data. 
+For example by setting the path to a 
+file "my-table.csv" in the directory "data", relative to your script file:
+    datafile <- file.path("data", "my-table.csv")
+Then you can for example read in the file using:
+    mytable <- read.csv(datafile) # Do something with that file name
+```
+
+The above Protip is very important, so make sure you understand this and use this within the geo-scripting exercises.
+-->
```

Comment:
Maybe good to think about ways to make the relative paths more clear to the students, because that's alwys a struggling point.